.husky/pre-push

```diff
@@ -1,3 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
What's the story on this and the `_` dir? Did a ctrl + f in the docs and didn't see an explanation.

Reply:
So that line gets generated automatically when using their `husky add` command when creating a new hook.

For the `_` dir, its added automatically on the initial install (`npm ci`)
![image](https://user-images.githubusercontent.com/25855114/115606829-36ab3380-a2b2-11eb-8e70-01ddcc1ad743.png)